src/serializer.js

```diff
@@ -915,7 +955,9 @@ var Serializer = (function () {
     }
     for (var ns in namespaceCounts) {
       if (!namespaceCounts.hasOwnProperty(ns)) continue
-      str += '\n xmlns:' + this.prefixes[ns] + '="' + escapeForXML(ns) + '"'
+      // Rel uris in xml ns is not strictly allowed in the spec but needed in practice often
+      var ns2 = (this.base && this.flags.indexOf('Z') < 0) ? Util.uri.refTo(this.base, ns) : ns
```

Comment:
Ok thanks for the explanation.

> We could offer a requested change to the spec at this late date o we could publish a set of spec changes assumed in solid.

The latter, at the least, sounds like a good idea. I think it's important that we document places where we "break" from the specs we use so that we can track our divergence.